tests/kernel/cache/testcase.yaml

```diff
@@ -8,6 +8,7 @@ tests:
       - qemu_xtensa
       - qemu_cortex_a53
       - nsim_em
+      - qemu_x86_64
```

Comment:
need to add `qemu_x86`

Reply:
I did not add qemu_x86 since it did not work (a new commit fixes it)